sled-hardware/src/lib.rs

```diff
@@ -66,13 +66,13 @@ impl std::fmt::Display for DendriteAsic {
     }
 }
 
-/// Configuration for forcing a sled to run as a Scrimlet or Gimlet
+/// Configuration for forcing a sled to run as a Scrimlet or compute Sled
 #[derive(Copy, Clone, Debug)]
 pub enum SledMode {
-    /// Automatically detect whether to run as a Gimlet or Scrimlet (w/ real Tofino ASIC)
+    /// Automatically detect whether to run as a compute sled or Scrimlet (w/ real Tofino ASIC)
     Auto,
     /// Force sled to run as a Gimlet
```

Comment:
This probably ought to be updated to say "non-switch-attached sled" or something. Unlike some of the other comment nits I've left, this one might actually be important; we don't want to create the impression that it forces a Cosmo to pretend to be a Gimlet or something...

Reply:
Yeah...see my other comment here. Btw, I proposed "swed" but backed off when people started looking at me with murderous intent.